### PR TITLE
Claimed amount on success screen

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Show claimed amount on success screen:

Screen Shot 2022-01-21 at 13 44 46
Screen Shot 2022-01-21 at 13 45 08

* Claimed amount is stored on redux as a locale aware formatted amount string
* It's saved after the claim callback returns (after user sends tx)

  # To Test

1. Make a claim
* Observe claimed amount of vCow is displayed while claiming and on success screen

### CODE REVIEW

**Comment by biocom** [comment]
Nice looks good! Will style further in upcoming PRs.

**Review by fairlighteth — APPROVED** [review]

File: src/custom/state/claim/reducer.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export type ClaimState = {
   isSearchUsed: boolean
   // claiming
   claimStatus: ClaimStatus
-  claimedAmount: number
+  claimedAmount: string
```
Comment (anxolin) [inline src/custom/state/claim/reducer.ts:56]: this is also derived from the state, although that was already the case before this PR. I guess we replicate for convenience, right?

**Review by anxolin — COMMENTED** [review]
Should we include also the amount while you are signing the transaction?

image

File: src/custom/state/claim/reducer.ts (hunk 1 above)
Comment (alfetopito) [reply src/custom/state/claim/reducer.ts:56]: It's a bit expensive/annoying to calculate.
The state has the amounts invested in the invested token, so we need to build the currencyamount objects and apply the prices to calculate the vCow amounts and them sum it up.

**Comment by alfetopito** [comment]
> Should we include also the amount while you are signing the transaction?
> 
> image

Due to lazyness and simplicity I decided not to.
Simplicity because I can re-use the calculation done on the claim callback and not have to re-do it everywhere else.

And because of that I don't have the amount until the tx is sent.

I could pass a callback to the claim callback but that does not sound like a good idea.

The alternative is to repeat the calculation outside of the claim callback.

I can do that if you think this is a must, but I don't think it's a big deal not having it until the tx is sent

File: src/custom/state/claim/reducer.ts (hunk 1 above)
Comment (anxolin) [reply src/custom/state/claim/reducer.ts:56]: okok

**Comment by anxolin** [comment]
> I can do that if you think this is a must, but I don't think it's a big deal not having it until the tx is sent

Not that bad. I think is nice to see while you sign it, but imo we can live without it

**Review by anxolin — APPROVED** [review]

**Review by W3stside — APPROVED** [review]
Seems good!

**Comment by elena-zh** [comment]
LGTM!

**Review by elena-zh — APPROVED** [review]

**Pull request merged**